osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-smr51.json

```diff
@@ -62,6 +62,90 @@
           "access": "RW"
         }
       ]
+    },
+    {
+      "tag": "ACTIVITY_CALENDAR",
```

Comment:
Waarom zit deze niet in 5.0.0??

Reply:
Deze hoort inderdaad in de SMR 5.0.0 te zittten, ik zal dit fixen.